Tests/Marketplace/upload_packs.py

```diff
@@ -0,0 +1,401 @@
+import json
```

Comment:
we really need to use type hints in all function signatures..

Reply:
each function has docstring with type of args and return value

Reply:
linters dont look at docstrings..